.github/workflows/dapr-release-notes.yml

```diff
@@ -0,0 +1,44 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+name: dapr-release-notes
+
+on: 
+  workflow_dispatch:
```

Comment:
is the `:` needed at the end?

Reply:
seems to work even with `:` ... https://github.com/artursouza/dapr/actions/runs/285569119/workflow